docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -8,29 +8,39 @@ Installing DIRAC client
 The DIRAC client installation procedure consists of few steps.
 You can do these steps as any user without the need to be root.
 
-A DIRAC client installation (and a server too) is fully in user space, and in fact, it's all in one directory,
-which means that on the same machine you can have several client(s) (or even server(s)) installed.
 
-If you want to create a shared client installation, you can do it by simply giving (UNIX) access
-to the directory where the client is installed.
+Python3 client installations
+----------------------------
 
+Choose the directory where you want to install the DIRAC client and::
 
-Install script (for python2 clients)
-------------------------------------
+   $ curl -LO https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-$(uname -m).sh
```

Comment:
Can you add
```
.. highlight:: console
``` somewhere before the first block, this should let one copy the commands without the $ in front, as used also here: https://dirac.readthedocs.io/en/latest/AdministratorGuide/Tutorials/diracSE.html
(reset to python if necessary in this document)

Reply:
Thanks, done.